src/pages/account/logins/[type]/new.tsx

```diff
@@ -43,6 +44,12 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     context,
     [withLogger, withSentry, withSession, withAuth],
     async () => {
+      if (context.params?.type?.toString() !== ("email" || "phone")) {
```

Comment:
this will not work

Reply:
`if (!["email", "phone"].includes(context.params?.type?.toString()))`

Reply:
Damn, I keep making this mistake..